tm2/pkg/p2p/node_info.go

```diff
@@ -12,6 +12,11 @@ const (
 	maxNumChannels  = 16    // plenty of room for upgrades, for now
 )
 
+const (
```

Comment:
should live outside p2p. prob tm2/pkg/bft/eventstore/types.

Reply:
Moved to the `eventstore` package:
[20e9a5c](https://github.com/gnolang/gno/pull/546/commits/20e9a5c2b98ff97f800b494af6c2492f11a2c6ea)